core/src/main/java/nl/nn/adapterframework/extensions/graphviz/GraphvizEngine.java

```diff
@@ -156,21 +155,22 @@ private String getVisJsWrapper() {
 
 		Engine(String initScript, String graphvisJsLibrary) {
 			// Available JS Engines. Lower index has priority.
-			Class<?>[] engines = new Class[]{J2V8.class, Nashorn.class};
+			String[] engines = AppConstants.getInstance().getString("javascript.engines", "nl.nn.adapterframework.extensions.javascript.J2V8,nl.nn.adapterframework.extensions.javascript.Nashorn").split(",");
```

Comment:
I think this should be done once, (moved to the top). Otherwise this will happen every time a new engine is generated..

You can, instead of using split, use `AppConstants.getTokenizedProperty()`